test/e2e/ci/ci_test.go

```diff
@@ -53,6 +58,26 @@ var _ = ginkgo.Describe("CI: Test for basic scheduling", func() {
 		gomega.Ω(err).NotTo(gomega.HaveOccurred())
 		gomega.Ω(ns1.Status.Phase).To(gomega.Equal(v1.NamespaceActive))
 
+		ginkgo.By("Deploy the example Application to the development namespace")
```

Comment:
To move this as separate spec, please use this documentation.... https://onsi.github.io/ginkgo/#getting-started-writing-your-first-test

Reply:
hi @HuangTing-Yao  is this done? I see you moved the file out, do we need to update 
```
ginkgo -r -v ci -timeout=2h -- -yk-namespace "yunikorn" -kube-config "$HOME/.kube/config"
```
as well to make sure this can be included in the test run?

Reply:
@yangwwei I am done, I separate the app part from ci suite.
I don't think we need to update that command,`make e2e_test` will run all suite, so it will cover the app part.